src/commands/synthetics/README.md

```diff
@@ -343,39 +486,380 @@ Your test files must be named with a `.synthetics.json` suffix.
-| `variables`                        | Object           | The variables to replace in the test. This object should contain key as the name of the variable to replace and values as the new value of the variable to replace. <br><br> With `--override`, use `variables.NAME=VALUE`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+#### `allowInsecureCertificates` (Boolean)
+
+Disable certificate checks in Synthetic API and Browser tests.
+
+Configuration options:
+
+* Test Config: `"allowInsecureCertificates": true`
+* ENV variable: `DATADOG_SYNTHETICS_OVERRIDE_ALLOW_INSECURE_CERTIFICATES=true`
+* CLI param: `--override allowInsecureCertificates=true`
+
+#### `basicAuth` (Object)
+
+Credentials to provide if basic authentication is required.
+
+* `username` (String): The username for basic authentication.
+* `password` (String): The password for basic authentication.
+
+Configuration options:
+
+* Test Config: `"basicAuth": {"username": "test_username", "password": "test_password"}`
+* ENV variable:
+  * `DATADOG_SYNTHETICS_OVERRIDE_BASIC_AUTH_USERNAME=test_username`
+  * `DATADOG_SYNTHETICS_OVERRIDE_BASIC_AUTH_PASSWORD=test_password`
+* CLI param:
+  * `--override basicAuth.username=test_username`
+  * `--override basicAuth.password=test_password`
+
+#### `body` (String)
+
+Data to send in an API test.
+
+Configuration options:
+
+* Test Config: `"body": "{\"fakeContent\":true}"`
+* ENV variable: `DATADOG_SYNTHETICS_OVERRIDE_BODY={"fakeContent":true}`
+* CLI param: `--override body={"fakeContent":true}`
+
+#### `bodyType` (String)
+
+Content type for the data to send in an API test.
+
+Configuration options:
+
+* Test Config: `"bodyType": "application/json"`
+* ENV variable: `DATADOG_SYNTHETICS_OVERRIDE_BODY_TYPE=application/json`
+* CLI param: `--override bodyType=application/json`
+
+#### `cookies` (String or object)
+
+Use the provided string as a cookie header in an API or browser test (in addition or as a replacement).
+
+* If this is a string, it is used to replace the original cookies as `append` defaults to `false`.
```

Comment:
Well this can be easily solved by just swapping the order of the 2 bullet points :D 
but i think that it does actually sound better as you suggest it and it doesn't need explain internal logic